tests/test_maplestory.py

```
@@ -82,10 +184,51 @@ def test_embed_creation():
+
+def test_edge_cases():
+    """測試邊緣情況"""
+    mock_bot = Mock()
+
+    # 測試空資料的情況
+    with patch.object(MapleStoryCogs, "_load_monsters_data", return_value=[]):
+        maple_cog = MapleStoryCogs(mock_bot)
+
+        # 搜尋應該返回空列表
+        monsters_found = maple_cog.search_monsters_by_name("測試")
+        assert monsters_found == [], "空資料時搜尋怪物應該返回空列表"
```

Pylint (3.3.7)

[convention] "monsters_found == []" can be simplified to "not monsters_found", if it is strictly a sequence, as an empty list is falsey

(C1803)